lib/elixir/lib/file.ex

```diff
@@ -1002,7 +1002,7 @@ defmodule File do
     F.open(IO.chardata_to_string(path), normalize_modes(modes, true))
   end
 
-  def open(path, function) when is_function(function) do
+  def open(path, function) when is_function(function, 1) do
     open(path, [], function)
   end
```

Comment:
yes. i know. but if you read the documentation and read the specs you are clueless